boxsdk/object/item.py

```diff
@@ -5,11 +5,56 @@
 import json
 
 from .base_object import BaseObject
+from boxsdk.config import API
```

Comment:
IMO, this is actually OK. Since 'item' module is in the 'object' package while apparently config module is not. Ideally, IMO, we would add 'from .config import API' to boxsdk/__init__.py and we can just do 'from boxsdk import API' here.